### PR TITLE
Handle zero hops separately if all travel times in feed are rounded

### PR DESCRIPTION
**Pull request by landonreed**
This PR implements the changes suggested by @abyrd in https://github.com/conveyal/gtfs-lib/pull/110#issuecomment-381582256.

cc @Tristramg

### CODE REVIEW

File: src/main/java/com/conveyal/gtfs/validator/SpeedTripValidator.java [hunk 1]
```diff
@@ -53,9 +57,13 @@ public void validateTrip(Trip trip, Route route, List<StopTime> stopTimes, List<
...
             }
-            // TODO detect the case where all travel times are rounded off to minutes (i.e. tt % 60 == 0)
-            // In that case determine the maximum and minimum possible speed by adding/removing one minute of slack.
+            // Detect if travel times are rounded off to minutes.
+            boolean bothTravelTimesRounded = areTravelTimesRounded(prevStopTime);
             double travelTimeSeconds = currStopTime.arrival_time - prevStopTime.departure_time;
+            // If travel times are rounded and travel time is zero, determine the maximum and minimum possible speed
+            // by adding/removing one minute of slack.
+            if (bothTravelTimesRounded && travelTimeSeconds == 0)
+                travelTimeSeconds += 60;
```
Comment (abyrd) [inline src/main/java/com/conveyal/gtfs/validator/SpeedTripValidator.java]: Please put brackets around conditional bodies unless there's no line break.

**Comment by abyrd** [comment]
@landonreed I approved the PR in principle (though there's request for a bracket change), and realized afterward that two snapshot tests are failing.

**Comment by landonreed** [comment]
Note: future work on rounded travel times should reference #157.

**Pull request merged**